app/src/main/java/com/sopt/gongbaek/presentation/ui/mypage/screen/MyPageViewModel.kt

```diff
@@ -88,4 +110,17 @@ class MyPageViewModel @Inject constructor(
             )
         }
     }
+
+    fun getVersionInfo(context: Context) {
+        viewModelScope.launch {
+            runCatching {
+                val packageInfo = context.packageManager.getPackageInfo(context.packageName, 0)
+                packageInfo.versionName
+            }.onSuccess { versionName ->
+                setState { copy(versionName = versionName) }
+            }.onFailure {
+                setState { copy(versionName = context.getString(R.string.setting_version_info_failed)) }
+            }
+        }
+    }
```

Comment:
[1] context 사용없이 바로 BuildConfig.VERSION_NAME 으로 접근해서 사용하는 방법이 더 좋다고 생각이 됩니다!
```suggestion
fun getVersionInfo() {
    val versionName = BuildConfig.VERSION_NAME
    setState { copy(versionName = versionName) }
}
```

추가로 앱 사용동안 업데이트를 하지 않으면 변경사유가 없어서 이 메서드도 날리고 그냥 state에 초기값으로 넣어두는건 어떤지? 의견을 남겨봅니다~!

Reply:
좋네요! 단순히 현재 앱의 버전만 표시할 목적이므로 바꾼 방식이 깔끔하고 좋을 것 같습니다.
1. Context 때문에 ViewModel이 Android Framework에 의존하게 되는 문제해결 
2. 비동기 처리(viewModelScope)가 아닌 동기 처리라 간단하고 깔끔한 코드

Reply:
좋은 것 같아요! 코드를 작성하면서도 context를 사용하지 않는 다른 방법은 없을까 고민했는데, BuildConfig를 사용하는 방법이 있었네요 👍👍 바로 수정하겠습니다 !